SharedSource/Development/UnitTesting/PrivateInvoke.cs

```diff
@@ -302,7 +304,7 @@ public static void SetNonPublicProperty (object target, Type declaringType, stri
     public static void SetPublicStaticProperty (Type type, string propertyName, object? value)
     {
       if (type == null) throw new ArgumentNullException ("type");
-      PropertyInfo property = type.GetProperty (propertyName, BindingFlags.Static | BindingFlags.Public);
+      PropertyInfo property = type.GetProperty (propertyName, BindingFlags.Static | BindingFlags.Public)!; // TODO: What's this?
```

Comment:
don't understand the question.

Reply:
`property` is unused, why bother having this line?

Reply:
I believe somebody was using the Windows clipboard while writing this type. The line may go and visit it's ancestors in the great unmanaged heap.

Reply:
(just maybe not in the NRT PR, to keep things readable)